### DISCUSSION TITLE
Shall we remove ribosomal genes during QC procedures?

### DISCUSSION THREAD

**Discussion by hyjforesight** [satijalab/seurat#5003] [Q&A] [answered]
Hello all,
In Seurat, we do QC to remove cells with high percentage of mitochondrial genes expressions.

pbmc[["percent.mt"]] <- PercentageFeatureSet(pbmc, pattern = "^MT-")
pbmc <- subset(pbmc, subset = nFeature_RNA > 200 & nFeature_RNA < 2500 & percent.mt < 5)

Do we need to remove ribosomal genes too?

Thanks!
Best,
YJ

**Comment by timoast** [comment, marked as answer]
Just a small clarification: in the code above you remove cells with high mitochondrial gene counts, not mitochondrial genes. I guess your question is whether we should filter cells based on the fraction of counts from ribosomal genes, not whether to remove ribosomal genes from the matrix (which you should not do).

You can certainly filter cells based on other QC metrics like the fraction of ribosomal gene counts, if you have reason to believe these metrics correlate with poor cell quality.

**Reply by hyjforesight** [reply]
Hello timoast, thanks for the reply.
Yes, that's what I mean. I was wondering whether it is necessary to remove cells with high percent expressions of ribosomal genes.
I have another strategy, but i'm not sure whether it is right. Because QC removes cells which may also remove small cell clusters, I did no QC before SCTranform. I just included all cells and do the 1st UMAP. Based on the clustered cell markers, I subset the clusters with no mitochondrial genes enriched and then do the 2nd UMAP.  Is this strategy right?
Thanks!
Best,
YJ
